.changeset/4s-use-rfr.md

```diff
@@ -0,0 +1,6 @@
+---
+issue: https://github.com/pmedianetwork/webpack-config/issues/4
+type: major
```

Comment:
I assume it's `major` because we need to enable `reactHotLoader` now manually if we want it, right?

Reply:
Exactly. It's an API breaking change.